tests/framework/AnalyticModels/optimizing/offset_parabola.py

```diff
@@ -49,3 +49,16 @@ def constrain(raven):
   if 0.25 < x < 0.75 and 0 < y < 1:
     return False
   return True
+
+def implicitConstrain(raven):
+  """
+    Implicit constrain.
+    @ In, raven, object, raven self
+    @ Out, implicit constrain, point ok or not?
```

Comment:
the standard requires, if not a local variable is created (like in this case), to specify in the docstrings the name of the function itself :

```
  @ Out, implicitConstrain, point ok or not?
```